### PR TITLE
remove preloading and cache wiping for test versions

### PR DESCRIPTION
**Pull request by rochdev**
### What does this PR do?

Remove preloading and cache wiping for test versions.

### Motivation

The previous behavior prevents relevant tests from running when the version is accidentally skipped.

### CODE REVIEW

File: packages/datadog-plugin-hapi/test/index.spec.js [hunk 1]
```diff
@@ -101,7 +101,8 @@ describe('Plugin', () => {
             expect(traces[0][0].meta).to.have.property('span.kind', 'server')
             expect(traces[0][0].meta).to.have.property('http.url', `http://localhost:${port}/user/123`)
             expect(traces[0][0].meta).to.have.property('http.method', 'GET')
-            expect(traces[0][0].meta).to.have.property('http.status_code', '200')
+            expect(traces[0][0].meta).to.have.property('http.status_code')
+            expect(Number(traces[0][0].meta['http.status_code'])).to.be.within(200, 299)
```
Comment (bengl) [inline packages/datadog-plugin-hapi/test/index.spec.js]: Why is this change here?

Comment (rochdev) [reply packages/datadog-plugin-hapi/test/index.spec.js]: `hapi` switched from `200` to `204` when the response is empty. For the purpose of the test, we don't care as long as it's a 2xx.
